boot/macros.h

```diff
@@ -12,7 +12,7 @@
 
 #ifndef __ASSEMBLY__
 
-#ifdef __GNUC__
+#if defined(__GNUC__) && !defined(__loongarch_lp64)
```

Comment:
Well... I took these enhanced definitions from a recent Linux kernel version, multiple releases after Loongarch support was added to the Linux kernel, so I'm surprised by this. Could you elaborate why memtest86+ shouldn't use them on LA64 ?

Reply:
Sorry, I forgot to change `c11` to `gnu11` in the Makefile, this patch will be dropped in next time.